.github/workflows/snapshot.yml

```diff
@@ -0,0 +1,153 @@
+name: Publish snapshots
+
+on:
+  workflow_call:
+    outputs:
+      avm-version:
+        description: "avm snapshot version"
+        value: ${{ jobs.publish-snapshot.outputs.avm-version }}
+
+env:
+  FORCE_COLOR: true
+
+jobs:
+  air-interpreter:
+    name: "Build air-interpreter wasm"
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - name: Checkout Aquavm
+        uses: actions/checkout@v3
+
+      - name: Setup Rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Install marine
+        uses: baptiste0928/cargo-install@v1
+        with:
+          crate: marine
```

Comment:
maybe use `download_marine.sh`? it's usually faster by an order or two

Reply:
![image](https://user-images.githubusercontent.com/28529663/190142106-ef482612-4335-4d0e-a620-f045c681e7ce.png) :)